Resources/Prototypes/XenoArch/Effects/normal_effects.yml

```
@@ -411,7 +411,7 @@
 
 - type: artifactEffect
   id: EffectCashSpawn
-  targetDepth: 2
+  targetDepth: 1
```

This was fine before - the big cash node should be moved to a higher depth instead
```suggestion
  targetDepth: 2
```